### PR TITLE
Fix indentation for all Swift files.

### PR DESCRIPTION
**Pull request by thai-d-v**
## Bugfix

I re-indent all Swift files (2 space width).

### CODE REVIEW

File: BookPlayer/AppDelegate.swift [hunk 1]
```diff
@@ -23,12 +23,12 @@ import WatchConnectivity
 class AppDelegate: UIResponder, UIApplicationDelegate {
   static weak var shared: AppDelegate?
   var pendingURLActions = [Action]()
-
```
Comment (GianniCarlo) [inline BookPlayer/AppDelegate.swift:26]: Is there a way to leave unmodified the empty lines without any whitespace?

Comment (thai-d-v) [reply BookPlayer/AppDelegate.swift:26]: I use the “Re-Indent” command in Xcode. Could you show me how you format your code? I will try to re-format it using your tools and settings. 😁

Comment (harrisi) [reply BookPlayer/AppDelegate.swift:26]: There are a few options, but the setting for this is in Settings (`⌘,`) > Text Editing > Editing > While Editing, and setting both the "Automatically trim trailing whitespace" and "Including whitespace-only lines".

Sometimes this doesn't work very reliably with re-indent, so you may need to find and replace using something like sed with something like `s/^\s+$//g`.

Comment (thai-d-v) [reply BookPlayer/AppDelegate.swift]: @GianniCarlo I fixed it. Thank you @harrisi.

**Review by GianniCarlo — APPROVED** [review]
Nice!

**Pull request merged**